README.md

```diff
@@ -1102,6 +1331,149 @@ Loading into XIP RAM: [==============================]  100%
+        -F, --force-no-reboot
+            Force a device not in BOOTSEL mode but running compatible code to reset so the command can be executed. After executing the
+            command (unless the command itself is a 'reboot') the device will be left connected and accessible to picotool, but without the
+            RPI-RP2 drive mounted
+```
+
+### list
+
+This command will list all the defined OTP rows & fields. You can list specific rows/fields using the selector
+
+```text
+$ picotool help otp list
+OTP LIST:
+    List matching known registers/fields
+
+SYNOPSIS:
+    picotool otp list [-p] [-n] [-f] [-i <filename>] [<selector>..]
+
+OPTIONS:
+    Row/Field Selection
+        -p, --pages
+            Show page number/page row number
+        -n, --no-descriptions
+            Don't show descriptions
+        -f, --field-descriptions
+            Show all field descriptions
+        -i <filename>
+            Include extra otp definition
+        <selector>
+            The row/field selector, each of which can select a whole row:
+
+            ROW_NAME to select a whole row by name.
+            ROW_NUMBER to select a whole row by number.
+            PAGE:PAGE_ROW_NUMBER to select a whole row by page and number within page.
+
+            ... or can select a single field/subset of a row (where REG_SEL is one of the above row selectors):
+
+            REG_SEL.FIELD_NAME to select a field within a row by name.
+            REG_SEL.n-m to select a range of bits within a row.
+            REG_SEL.n to select a single bit within a row.
+            .FIELD_NAME to select any row's field by name.
+
+            .. or can selected multiple rows by using blank or '*' for PAGE or PAGE_ROW_NUMBER
+```
+
+## coprodis
+
+This command is used by the SDK to post-process dissassembly files, to replace the coprocessor instructions (mcr, mrc, etc.) with more human readable ones (eg rcp_bvalid, gpioc_hi_put, etc.).
+
+```text
+$ picotool help coprodis
+COPRODIS:
+    Post-process coprocessor instructions in disassembly files.
+
+SYNOPSIS:
+    picotool coprodis [--quiet] [--verbose] <infile> [-t <type>] <outfile> [-t <type>]
+
+OPTIONS:
+        --quiet
+            Don't print any output
+        --verbose
+            Print verbose output
+    Input DIS
+        <infile>
+            The file name
+        -t <type>
+            Specify file type (uf2 | elf | bin) explicitly, ignoring file extension
+    Output DIS
+        <outfile>
+            The file name
+        -t <type>
+            Specify file type (uf2 | elf | bin) explicitly, ignoring file extension
+```
+
+## link
+
+This command is used to link multiple binaries with block loops into a single larger block loop. It can currently link up to 3 files into a block loop. It will add the required Rolling Window Delta items to the new block loop, to ensure that everyting is rolled correctly when being executed. For examples of it's usage, see the universal examples in pico-examples.
```

Comment:
nit: `it's` -> `its`

I guess there'd be no harm in adding a direct link to https://github.com/raspberrypi/pico-examples?tab=readme-ov-file#universal here?